invenio_rdm_records/resources/serializers/__init__.py

```diff
@@ -34,6 +35,6 @@
     "IIIFManifestV2JSONSerializer",
     "IIIFSequenceV2JSONSerializer",
     "MARCXMLSerializer",
-    "StringCitationSerializer",
+    "BibtexSerializer",
```

Comment:
major: should add, not replace. Also, we dont enforce but it is nice if they are ordered alphabetically.